### DISCUSSION TITLE
Zim wiki (0.74.3) doesn't support Tamil language

### DISCUSSION THREAD

**Discussion by dhana25252** [zim-desktop-wiki/zim-desktop-wiki#2218] [Q&A] [answered]
Hello there (Developers),

I'm a newbie to GitHub, I've been using Zim Wiki Desktop for several years, experiencing excellent application!!

Zim Wiki is not supporting Tamil Fonts, please let me know is there any way to use Tamil Fonts,
Attached snap below.

Regards,
Dhana V
dhana.varadaraju@gmail.com
![image](https://user-images.githubusercontent.com/13852137/198983990-e9da06a7-af27-441d-8cd9-7064deec3a5b.png)

**Comment by jaap-karssenberg** [comment]
Afraid this is not a Zim specific issue. Fonts are supported by the GTK
framework. I'm pretty sure there must be a way to install it select a Tamil
font, since GTK has good localization support. But can't give you direct
instructions. Please consult the internet.

Op ma 31 okt. 2022 11:11 schreef dhana25252 ***@***.***>:

**Reply by dhana25252** [reply]
Thanks a lot for the prompt response, my issue was resolved.

**Comment by dehesselle** [comment, marked as answer]
I'm able to copy-paste Tamil script into Zim:

<img width="500" alt="image" src="https://user-images.githubusercontent.com/11375494/199018328-cfbc7a9a-24cc-4019-9359-457843a5aa31.png">

Looking into the file, I can even see it correctly in the terminal. So first thing for you to check is that your OS can handle this correctly.

**Reply by dhana25252** [reply]
Thanks a lot for the post, I can able to see the Tamil font now, thanks once again!! :-)
